docfx_project/articles/bootstrapping-trust/part-01.encryption-everywhere.md

```diff
@@ -0,0 +1,207 @@
+# Bootstrapping Trust
+
+## Part 1: Encryption Everywhere
+
+### Zero Trust
+
+Bootstrapping Trust is a series highlighting a common problem dealing
+with software systems that span multiple machines over any network. All
+networks, private (like your office/data center) or public (like the
+internet), should be considered insecure. Firewalls and segmented
```

Comment:
This sentence doesn't flow. `Network professionals have segmenting networks and started praising application-specific networking, but even those aren't enough.`

Might be trying to say too much there. Maybe just `Even application-specific networking is not enough.`. You've already covered segmented networking in the previous sentence.

Reply:
Mmm updated.